docs/source/en/using-diffusers/controlling_generation.mdx

```diff
@@ -27,108 +27,121 @@ Depending on the use case, one should choose a technique accordingly. In many ca
 
 Similar to how classifier free guidance provides guidance via empty prompt inputs, SEGA provides guidance on conceptual prompts. Multiple of these conceptual prompts can be applied simultaneously. Each conceptual prompt can either add or remove their concept depending on if the guidance is applied positively or negatively.
 
+Unlike Pix2Pix Zero or Attend and Excite, SEGA directly interacts with the diffusion process instead of performing any explicit gradient-based optimization. 
+
 See [here](../api/pipelines/semantic_stable_diffusion) for more information on how to use it.
 
-## Self attention guidance
+## Self-attention Guidance (SAG)
 
 [Paper](https://arxiv.org/abs/2210.00939)
 
-[Self attention guidance](../api/pipelines/stable_diffusion/self_attention_guidance) improves the general quality of images.
+[Self-attention Guidance](../api/pipelines/stable_diffusion/self_attention_guidance) improves the general quality of images.
 
-SAG provides guidance from predictions not conditioned on high frequency details to fully conditioned images. The high frequency details are extracted out of the UNet self-attention maps.
+SAG provides guidance from predictions not conditioned on high-frequency details to fully conditioned images. The high frequency details are extracted out of the UNet self-attention maps.
 
 See [here](../api/pipelines/stable_diffusion/self_attention_guidance) for more information on how to use it.
 
-## Depth2image
+## Depth2Image
 
-[Paper](https://huggingface.co/stabilityai/stable-diffusion-2-depth)
+[Project](https://huggingface.co/stabilityai/stable-diffusion-2-depth)
 
-[Depth2image](../pipelines/stable_diffusion_2#depthtoimage) is fine-tuned from stable diffusion to better preserve semantics for text guided image variation. 
+[Depth2Image](../pipelines/stable_diffusion_2#depthtoimage) is fine-tuned from Stable Diffusion to better preserve semantics for text guided image variation. 
 
 It conditions on a monocular depth estimate of the original image.
 
-
 See [here](../api/pipelines/stable_diffusion_2#depthtoimage) for more information on how to use it.
 
-### Fine-tuning methods
+<Tip>
+
+An important distinction between methods like InstructPix2Pix and Pix2Pix Zero is that the former
+involves fine-tuning the pre-trained weights while the latter does not.
+
+</Tip>
+
+## MultiDiffusion Panorama
+
+[Paper](https://arxiv.org/abs/2302.08113)
+
+MultiDiffusion defines a new generation process over a pre-trained diffusion model. This process binds together multiple diffusion generation processes can be readily applied to generate high quality and diverse images that adhere to user-provided controls, such as desired aspect ratio (e.g., panorama), and spatial guiding signals, ranging from tight segmentation masks to bounding boxes.
```

Comment:
```suggestion
MultiDiffusion defines a new generation process over a pre-trained diffusion model. This process binds together multiple diffusion generation methods that can be readily applied to generate high quality and diverse images. Results adhere to user-provided controls, such as desired aspect ratio (e.g., panorama), and spatial guiding signals, ranging from tight segmentation masks to bounding boxes.
```

Reply:
(Question: can our pipeline be used for segmentation or just panorama?)

Reply:
Not yet. Cc: @omerbt